Exiled.CustomRoles/API/Features/CustomRole.cs

```diff
@@ -528,18 +534,18 @@ public virtual void AddRole(Player player)
                         Log.Debug($"{Name}: Adding {itemName} to inventory.");
                         TryAddItem(player, itemName);
                     }
-                });
 
-            Log.Debug($"{Name}: Setting health values.");
-            player.Health = MaxHealth;
-            player.MaxHealth = MaxHealth;
-            player.Scale = Scale;
+                    Log.Debug($"{Name}: Setting health values.");
+                    player.Health = MaxHealth;
+                    player.MaxHealth = MaxHealth;
+                    player.Scale = Scale;
 
-            Vector3 position = GetSpawnPosition();
-            if (position != Vector3.zero)
-            {
-                player.Position = position;
-            }
+                    Vector3 position = GetSpawnPosition();
+                    if (position != Vector3.zero)
+                    {
+                        player.Position = position;
+                    }
+                });
```

Comment:
you don't need to delay all this stuff

Reply:
Ok, I'll close my or, let you remove them, and say I told you so when everything breaks. Have fun.